tests/test-maximize.lua

```diff
@@ -82,10 +82,11 @@ local steps = {
         local bw      = c.border_width
 
         assert(c.fullscreen)
-        assert(new_geo.x-bw==sgeo.x)
-        assert(new_geo.y-bw==sgeo.y)
-        assert(new_geo.x+new_geo.width+bw==sgeo.x+sgeo.width)
-        assert(new_geo.y+new_geo.height+bw==sgeo.y+sgeo.height)
+
+        assert(new_geo.x==sgeo.x)
+        assert(new_geo.y==sgeo.y)
+        assert(new_geo.x+new_geo.width+2*bw==sgeo.x+sgeo.width)
+        assert(new_geo.y+new_geo.height+2*bw==sgeo.y+sgeo.height)
```

Comment:
...so, does that mean that the next issue will be the "fullscreen" is no longer fullscreen? That's not good. Fullscreen should hide the border (=set border width to zero).